adafruit_mcp9808.py

```diff
@@ -74,26 +107,33 @@ class MCP9808:
 
     """
 
-    # alert_lower_temperature_bound
-    # alert_upper_temperature_bound
-    # critical_temperature
-    # temperature
-    # temperature_resolution
+    _MCP9808_REG_RESOLUTION_SET = RWBits(2, 0x08, 0, register_width=2)
+    above_ct = ROBit(_MCP9808_REG__TEMP, 7, register_width=1)
+    """True when the temperature is above than the currently
```

Comment:
```suggestion
    """True when the temperature is above the currently
```

Another correction is "higher than" instead of "above than".